pkg/data/components/FilterProcessor.yaml

```diff
@@ -28,10 +28,10 @@ ports:
 properties:
   - name: Rules
     type: rule
-    validations: [rule]
+    validations: [filterrule]
   - name: Signal
     type: string
-    validations: [string] # TODO: validate a valid signal
+    validations: ["oneof(traces, metrics, logs)"] # TODO: validate a valid signal
```

Comment:
We could add a custom validSignal rule that interally is made up using `oneof(traces, metrics, logs)`

Reply:
yes; let's see if we think it's common enough to justify. The presence of HoneycombEvents complicates this a bit.

Reply:
On review, I don't think we need this. There are going to be a mix of components with different support signals. Having an abstract one doesn't make it any clearer / easier to read.